sonar-db/src/main/gen-java/org/sonar/db/protobuf/DbCommons.java

```diff
@@ -0,0 +1,818 @@
+      public boolean hasStartOffset() {
+        return ((bitField0_ & 0x00000004) == 0x00000004);
+      }
+      /**
+       * <code>optional int32 start_offset = 3;</code>
+       *
+       * <pre>
+       * If absent it means range starts at the first offset of start line
+       * </pre>
+       */
+      public int getStartOffset() {
+        return startOffset_;
+      }
+      /**
+       * <code>optional int32 start_offset = 3;</code>
+       *
+       * <pre>
+       * If absent it means range starts at the first offset of start line
+       * </pre>
+       */
+      public Builder setStartOffset(int value) {
+        bitField0_ |= 0x00000004;
+        startOffset_ = value;
+        onChanged();
+        return this;
+      }
+      /**
+       * <code>optional int32 start_offset = 3;</code>
+       *
+       * <pre>
+       * If absent it means range starts at the first offset of start line
+       * </pre>
+       */
+      public Builder clearStartOffset() {
+        bitField0_ = (bitField0_ & ~0x00000004);
+        startOffset_ = 0;
+        onChanged();
+        return this;
+      }
+
+      private int endOffset_ ;
+      /**
+       * <code>optional int32 end_offset = 4;</code>
+       *
+       * <pre>
+       * If absent it means range ends at the last offset of end line
+       * </pre>
+       */
+      public boolean hasEndOffset() {
+        return ((bitField0_ & 0x00000008) == 0x00000008);
+      }
+      /**
+       * <code>optional int32 end_offset = 4;</code>
+       *
+       * <pre>
+       * If absent it means range ends at the last offset of end line
+       * </pre>
+       */
+      public int getEndOffset() {
+        return endOffset_;
+      }
+      /**
+       * <code>optional int32 end_offset = 4;</code>
+       *
+       * <pre>
+       * If absent it means range ends at the last offset of end line
+       * </pre>
+       */
+      public Builder setEndOffset(int value) {
+        bitField0_ |= 0x00000008;
+        endOffset_ = value;
+        onChanged();
+        return this;
+      }
+      /**
+       * <code>optional int32 end_offset = 4;</code>
+       *
+       * <pre>
+       * If absent it means range ends at the last offset of end line
+       * </pre>
+       */
+      public Builder clearEndOffset() {
+        bitField0_ = (bitField0_ & ~0x00000008);
+        endOffset_ = 0;
+        onChanged();
+        return this;
+      }
+
+      // @@protoc_insertion_point(builder_scope:sonarqube.db.commons.TextRange)
+    }
+
+    static {
+      defaultInstance = new TextRange(true);
+      defaultInstance.initFields();
+    }
+
+    // @@protoc_insertion_point(class_scope:sonarqube.db.commons.TextRange)
+  }
+
+  private static final com.google.protobuf.Descriptors.Descriptor
+    internal_static_sonarqube_db_commons_TextRange_descriptor;
+  private static
+    com.google.protobuf.GeneratedMessage.FieldAccessorTable
+      internal_static_sonarqube_db_commons_TextRange_fieldAccessorTable;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "internal_static_sonarqube_db_commons_TextRange_fieldAccessorTable" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS3008)